### DISCUSSION TITLE
Does speech recognition support Chinese? When does the VAD model support Chinese?

### DISCUSSION THREAD

**Discussion by DidaDidaDidaD** [speechbrain/speechbrain#1638] [Q&A]
Does speech recognition support Chinese? When does the VAD model support Chinese? Or how to train Chinese VAD model? If it can directly support the automatic segmentation and recognition of long speech, and give the start and end time information of segmented speech at the same time。Looking forward to your reply,thank you

**Comment by Adel-Moumen** [comment]
Hello @DidaDidaDidaD,

Yes, we do support Chinese for speech recognition. You can see an example [here](https://github.com/speechbrain/speechbrain/tree/develop/recipes/AISHELL-1/ASR). 

Maybe @BenoitWang will be able to answer on the VAD support of the Chinese language :-)

**Comment by BenoitWang** [comment]
Hi @DidaDidaDidaD @Adel-Moumen , sorry I have no idea about this... However, vad is less language dependant so maybe you can first test the model pretrained in English. If you want to train with a chinese dataset, just follow the LibriParty recipe and prepare your own data.

**Comment by mravanelli** [comment]
Hi,
as for speech recognition we have recipes with AISHELL.
As for the VAD, it is trained in English but I think it can perform well in
any other languages anyway.

Best,

Mirco
